sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -964,6 +964,21 @@ if (contextVersion > 1) {
 	    return gl.getActiveUniforms(program, uniformIndices, pname);
         }
     );
+
+    debug("");
+    debug("Test getUniformBlockIndex");
+    var program = wtu.loadUniformBlockProgram(gl);
+    gl.linkProgram(program);
+    shouldBeTrue('gl.getProgramParameter(program, gl.LINK_STATUS)');
+    shouldBe('gl.getUniformBlockIndex(program, "Transform")', '0');
```

Comment:
"Indices of the active uniform blocks of a program are assigned in consecutive order, beginning with zero. "
https://www.khronos.org/opengles/sdk/docs/man3/html/glGetUniformBlockIndex.xhtml
There is only one uniform block in 'program', so the index should be '0'.